### DISCUSSION TITLE
Tracking deadlocks

### DISCUSSION THREAD

**Discussion by clhambone** [trimble-oss/dba-dash#1025] [Ideas]
I was just curious if there was any thoughts on tracking deadlocks in the future.  I have reviewed to see if there is anything currently but I am unable to find anything so that is why I am asking.

**Comment by DavidWiseman** [comment]
There is a performance counter to track deadlocks.  It's displayed in deadlocks/sec which isn't ideal but it can show when deadlocks are occurring on your monitored instances.  

There is an issue already created here #67 for adding improved deadlock tracking.  I'll close this as a duplicate but feel free to comment on #67 if you have any additional thoughts on this.  Thanks.

**Comment by clhambone** [comment]
Thanks.
